metadata-ingestion/src/datahub/ingestion/source/looker/lookml_source.py

```diff
@@ -814,41 +837,95 @@ def get_refinement_from_model_includes(self, view_name: str) -> List[dict]:
                 continue
 
             refined_views.extend(
-                self.get_refinement_from_looker_view_file(
-                    included_looker_viewfile, view_name
+                self.get_refinement_from_view_list(
+                    included_looker_viewfile.views, view_name
                 )
             )
 
         return refined_views
 
+    def skip_processing(self, raw_view_name: str) -> bool:
```

Comment:
please also add a docstring to this method
```suggestion
    def should_skip_processing(self, raw_view_name: str) -> bool:
```

Reply:
done